Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include "MantidNexusGeometry/NexusGeometrySave.h"
 #include "MantidGeometry/Instrument/ComponentInfo.h"
+#include "MantidGeometry/Instrument/ComponentInfoBankHelpers.h"
```

Comment:
This include only makes sense if you move the functionality you just wrote into the helpers. At present, you do not use any of the functionality included.